crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -309,8 +309,8 @@ pub struct MeshPipeline {
     // This dummy white texture is to be used in place of optional StandardMaterial textures
     pub dummy_white_gpu_image: GpuImage,
     pub clustered_forward_buffer_binding_type: BufferBindingType,
-
     pub mesh_layouts: MeshLayouts,
+    pub mesh_buffer_batch_size: Option<u32>,
```

Comment:
I'd like to see a small comment explaining this  needs to be added as `ShaderDefVal::UInt("MESH_BATCH_SIZE"`

Reply:
Done.